spec/features/dams_object_spec.rb

```diff
@@ -841,6 +841,7 @@
     @unit.delete
   end
   it "should show a 'View file' button and a 'Download file' button " do
+    sign_in_curator
```

Comment:
Would it be possible to also add a test demonstrating that a vrr user explicitly cannot see/execute the `Download file` button? This sort of does it implicitly, but it might be nice to have an explicit test as well.

Reply:
@mcritchlow this is actually the old test unit forgot adding sign_in_curator, so I fixed here.
The vrr access only works on staging and production with token and user email. I'm not sure if it is possible to write the test.  I'm gonna dig into it.

Reply:
Sounds good!